ide-support/revsaveasiosstandalone.livecodescript

```diff
@@ -618,14 +618,19 @@ private command revSaveAsMobileStandaloneMain pStack, pAppBundle, pTarget, pSett
       put revGetMobileProfileInfo(pSettings["ios,profile"]) into tProfileInfo
       
       -- Get the list of valid identities that are available
-      local tValidIdentities
+      local tValidIdentities,tValidIdentitiesHEX
       get shell("/usr/bin/security -q find-identity -v")
```

Comment:
@livecodepanos this can further filter with a policy if we want although it probably doesn't make much difference:
```
get shell("/usr/bin/security -q find-identity -v -p codesigning")
```